modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/src/Form/WebtoolsAnalyticsRuleForm.php

```diff
@@ -40,6 +40,13 @@ public function form(array $form, FormStateInterface $form_state): array {
       '#disabled' => !$rule->isNew(),
     ];
 
+    $form['match_on_site_default_language'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Match on path alias for site default language.'),
+      '#default_value' => $rule->matchOnSiteDefaultLanguage(),
+      '#description' => $this->t("If you select this checkbox, regexp would be applied for all aliases. E.g.: if you have /news/ regexp value, rule could be applied for /fr/nouvelles/* and /nl/nieuws/* pathes."),
```

Comment:
This needs to be reworded too. Something on the line of `If checked, the matching will be done on the url alias for the site default language. E.g.: <something else>`.

Reply:
Fixed